main/mainworker.cpp

```diff
@@ -12076,7 +12076,8 @@ void MainWorker::HandleLogNotifications()
 	else
 	{
 		itt = _loglines.begin();
-		sTopic = "Domoticz: " + itt->logmessage;
+		sTopic = "Domoticz: Error received";
```

Comment:
they i rather not merge this request
if there are multiple errors in a minute, you will get another header and a body that contains all errors
I dont have a problem with the headers, using a normal 1920x1080 screen, not 2k or 4k